elasticsearch-scylla/README.md

```diff
@@ -0,0 +1,96 @@
+# Mine data with Elasticsearch and ScyllaDB
+<p align=center>
+
+The purpose of this demo is to show how to feed data from Twitter into Elasticsearch and Scylla for data analytics purposes. The Twitter app will search for a specific Twitter hashtag and feed data into Scylla. To feed the data from Scylla to Elasticsearch, simply issue a curl command referenced in this README to begin the data dump.
+
+### Architecture
+
+##### Scylla Architecture
+Three Scylla 1.7 nodes
+
+##### Elasticsearch Architecture
+Two Elasticsearch nodes (Master, data).
+![Pic](https://raw.githubusercontent.com/scylla/scylla-code-samples/master/elasticsearch-scylla/diagram.png)
+
+### Prerequisites
+
+1. Docker for [Mac](https://download.docker.com/mac/stable/Docker.dmg) or [Windows](https://download.docker.com/win/stable/InstallDocker.msi).
+2. This Git repository
+3. 3GB of RAM or greater for Docker
+4. Twitter API credentials. After signing up for a developer [account](https://dev.twitter.com/), go to the Twitter Apps [page](https://apps.twitter.com/) and click on “Create New App”. The API credentials are provided after the application is created. 
```

Comment:
Is the signing up for a developer [account](https://dev.twitter.com/) a must?
I only created a new apps and it seems to work